control/mpc_lateral_controller/src/mpc_lateral_controller.cpp

```diff
@@ -243,7 +243,9 @@ trajectory_follower::LateralOutput MpcLateralController::run(
   Trajectory predicted_traj;
   Float32MultiArrayStamped debug_values;
 
-  if (!m_is_ctrl_cmd_prev_initialized) {
+  if (
+    !m_is_ctrl_cmd_prev_initialized ||
+    !input_data.current_operation_mode.is_autoware_control_enabled) {
```

Comment:
should be like this ` !(is_autoware_control_enabled && mode==AUTONOMOUS)`?

Reply:
Thanks! I addressed in https://github.com/autowarefoundation/autoware.universe/pull/6541/commits/0516f5acf4233697dddc372708ce986606b8f949